HHS/Head-Start-TTADP
# [PROD] 06/20/24 TTAHUB-2667: Fix save pdf white space, TTAHUB-2519: Show Fei root cause per recipient

## Description of change

**TTAHUB-2667:** Fix for saving reports as PDF in windows.
**TTAHUB-2519:** We now show FEI root causes per recipient.
**TTAHUB-2132:** Add " - Active" and " - Inactive" to grant number filter on RTR.

## How to test

**TTAHUB-2667:** Saving PDF's in windows shouldn't have any white space after 'Activity dates'.
**TTAHUB-2519:** Reviewing an FYI report should show root causes per recipient.
**TTAHUB-2132:** Ensure the list looks something like this:

<img width="805" alt="Screenshot 2024-06-20 at 12 33 49 PM" src="https://github.com/HHS/Head-Start-TTADP/assets/17074357/2b81d5fb-6c55-4978-8c84-5a8c65d3c102">


## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2667
* https://jira.acf.gov/browse/TTAHUB-2519
* https://jira.acf.gov/browse/TTAHUB-2132


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
